src/content/insertable-streams/video-processing/js/simple-transforms.js

```diff
@@ -8,6 +8,21 @@
 
 'use strict';
 
+/**
+ * Does nothing.
+ * @implements {FrameTransform} in pipeline.js
+ */
+class NullTransform { // eslint-disable-line no-unused-vars
+  /** @override */
+  async init() {}
+  /** @override */
+  async transform(frame, controller) {
+    controller.enqueue(frame);
+  }
+  /** @override */
+  destroy() {}
+}
+
 /**
  * Drops frames at random.
```

Comment:
We might consider removing DropTransform in favor of NullTransform, since they don't really demonstrate unique uses of the API. (I just thought NullTransform would be too dull, so I tried to make it more interesting; however, NullTransform is more interesting for testing and experimentation.)